react_on_rails.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |s|
   s.add_dependency "addressable"
   s.add_dependency "connection_pool"
   s.add_dependency "execjs", "~> 2.5"
+  s.add_dependency "sqlite3", "~> 1.6"
```

Comment:
why make this a dependency? it is needed for tests only?

Reply:
Correct. My mistake.